spring-cloud-dataflow-audit/src/main/java/org/springframework/cloud/dataflow/audit/service/AuditServiceUtils.java

```diff
@@ -29,21 +33,51 @@
  *
  */
 public class AuditServiceUtils {
+	public static final String APP_NAME = "APP_NAME";
+
+	public static final String APP_TYPE = "APP_TYPE";
+
+	public static final String APP_VERSION = "APP_VERSION";
+
+	public static final String APP_IS_DEFAULT = "APP_IS_DEFAULT";
 
 	public static final String STREAM_DEFINITION_DSL_TEXT = "streamDefinitionDslText";
 
-	public static final String TASK_DEFINITION_DSL_TEXT = "taskDefinitionDslText";
 	public static final String TASK_DEFINITION_NAME = "taskDefinitionName";
+
 	public static final String TASK_DEFINITION_PROPERTIES = "taskDefinitionProperties";
 
 	public static final String DEPLOYMENT_PROPERTIES = "deploymentProperties";
+
 	public static final String COMMANDLINE_ARGUMENTS = "commandlineArguments";
 
 	private final ArgumentSanitizer argumentSanitizer = new ArgumentSanitizer();
 
 	public AuditServiceUtils() {
 	}
 
+	public Map<String, Object> convertAppRegistrationToAuditData(String name, ApplicationType type, String version) {
+		try {
+			return this.convertAppRegistrationToAuditData(new AppRegistration(name, type, version, new URI(""), null));
```

Comment:
I think app `uri` and `metadataUri` are the most important ones which are now not added.